README.md

```diff
@@ -1,3 +1,38 @@
-# Blackhole.service for DEIS
+# Blackhole service for Deis
 
-This will be a repository for dockerized blackhole service for DEIS.
+A repository for a docker container with simple Blackhole implementation for Deis (CoreOS) platform.
+
+The script bans every failed attempt to login by password on CoreOS instances except ip addresses that are included in `WHITELIST` variable.
+
+In example below `WHITELIST` includes all CoreOS instances in cluster.
```

Comment:
I would move this below the snippet and rewrite as

In this example `WHITELIST` variable points to all instance's ip addresses in a cluster.